app/View/Elements/genericElements/IndexTable/index_table.ctp

```diff
@@ -24,8 +24,8 @@
     if (!empty($data['html'])) {
         echo sprintf('<p>%s</p>', $data['html']);
     }
-    $skipPagination = isset($data['skip_pagination']) && $data['skip_pagination'];
-    if (!$skipPagination) {
+    $skip_pagination = isset($data['skip_pagination']) ? $data['skip_pagination'] : 0;
```

Comment:
this was fixed in a different PR since then, should also not really be part of this PR.